test/e2e/specs/editor/blocks/table.spec.js

```diff
@@ -0,0 +1,276 @@
+
+		// Expect the table to have 2 columns across the header, body and footer.
+		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+	} );
+
+	test( 'allows columns to be aligned', async ( { editor, page } ) => {
+		await editor.insertBlock( { name: 'core/table' } );
+
+		await page.click( 'role=spinbutton[name="Column count"i]' );
+		await page.keyboard.press( 'Backspace' );
+		await page.keyboard.type( '4' );
+
+		// Create the table.
+		await page.click( 'role=button[name="Create Table"i]' );
+
+		// Click the first cell and add some text. Don't align.
+		const cells = await page.locator(
+			'role=textbox[name="Body cell text"]'
+		);
+		await cells.nth( 0 ).click();
+		await page.keyboard.type( 'None' );
+
+		// Click to the next cell and add some text. Align left.
+		await cells.nth( 1 ).click();
+		await page.keyboard.type( 'To the left' );
+		await editor.clickBlockToolbarButton( 'Change column alignment' );
+		await page.click( 'role=menuitemradio[name="Align column left"]' );
+
+		// Click the next cell and add some text. Align center.
+		await cells.nth( 2 ).click();
+		await page.keyboard.type( 'Centered' );
+		await editor.clickBlockToolbarButton( 'Change column alignment' );
+		await page.click( 'role=menuitemradio[name="Align column center"]' );
+
+		// Tab to the next cell and add some text. Align right.
+		await cells.nth( 3 ).click();
+		await page.keyboard.type( 'Right aligned' );
+		await editor.clickBlockToolbarButton( 'Change column alignment' );
+		await page.click( 'role=menuitemradio[name="Align column right"]' );
+
+		// Expect the post to have the correct alignment classes inside the table.
+		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+	} );
+
+	// Testing for regressions of https://github.com/WordPress/gutenberg/issues/14904.
+	test( 'allows cells to be selected when the cell area outside of the RichText is clicked', async ( {
+		editor,
+		page,
+	} ) => {
+		await editor.insertBlock( { name: 'core/table' } );
+		await editor.openDocumentSettingsSidebar();
+
+		// Create the table.
+		await page.click( 'role=button[name="Create Table"i]' );
+
+		// Enable fixed width as it exascerbates the amount of empty space around the RichText.
+		await page
+			.locator( 'role=checkbox[name="Fixed width table cells"i]' )
+			.check();
+
+		// Add multiple new lines to the first cell to make it taller.
+		await page.click( 'role=textbox[name="Body cell text"] >> nth=0' );
+		await page.keyboard.type( '\n\n\n\n' );
+
+		// Get the bounding client rect for the second cell.
+		const { x: secondCellX, y: secondCellY } = await page.evaluate( () => {
+			const secondCell =
+				document.querySelectorAll( '.wp-block-table td' )[ 1 ];
+			// Page.evaluate can only return a serializable value to the
+			// parent process, so destructure and restructure the result
+			// into an object.
+			const { x, y } = secondCell.getBoundingClientRect();
+			return { x, y };
+		} );
```

Comment:
There's an API for [`boundingBox`](https://playwright.dev/docs/api/class-locator), could we use it here?

```suggestion
		const { x: secondCellX, y: secondCellY } = await page
			.locator( 'role=textbox[name="Body cell text"] >> nth=1' )
			.boundingBox();
```